change_color.sh

```diff
@@ -6,113 +6,118 @@ set -ueo pipefail
-  echo "       $0 --hidpi True ../colors/retro/clearlooks"
-  exit 1
+    echo "usage: $0 [-t TARGET_DIR] [-o OUTPUT_THEME_NAME] [-p PATH_LIST] PATH_TO_PRESET"
+    echo "examples:"
+    # shellcheck disable=SC2028 # This is meant to be usage text.
+    echo "       $0 -o my-theme-name <(echo -e \"ROUNDNESS=0\\nBG=d8d8d8\\nFG=101010\\nHDR_BG=3c3c3c\\nHDR_FG=e6e6e6\\nSEL_BG=ad7fa8\\nMATERIA_VIEW=ffffff\\nMATERIA_SURFACE=f5f5f5\\nMATERIA_STYLE_COMPACT=True\\n\")"
+    echo "       $0 -t ~/.themes ../colors/retro/twg"
+    echo "       $0 --hidpi True ../colors/retro/clearlooks"
+    exit 1
 }
 
 
 while [[ "$#" -gt 0 ]]; do
-  case "$1" in
-    -p|--path-list)
-      CUSTOM_PATHLIST="$2"
-      shift
-      ;;
-    -t|--target)
-      TARGET_DIR="$2"
-      shift
-      ;;
-    -o|--output)
-      OUTPUT_THEME_NAME="$2"
-      shift
-      ;;
-    -d|--hidpi)
-      OPTION_GTK2_HIDPI="$2"
-      shift
-      ;;
-    -i|--inkscape)
-      OPTION_FORCE_INKSCAPE="$2"
-      shift
-      ;;
-    *)
-      if [[ "$1" == -* ]] || [[ "${THEME-}" ]]; then
-        echo "unknown option $1"
-        print_usage
-        exit 2
-      fi
-      THEME="$1"
-      ;;
-  esac
-  shift
+    case "$1" in
+        -p|--path-list)
+            CUSTOM_PATHLIST="$2"
+            shift
+            ;;
+        -t|--target)
+            TARGET_DIR="$2"
+            shift
+            ;;
+        -o|--output)
+            OUTPUT_THEME_NAME="$2"
+            shift
+            ;;
+        -d|--hidpi)
+            OPTION_GTK2_HIDPI="$2"
+            shift
+            ;;
+        -i|--inkscape)
+            OPTION_FORCE_INKSCAPE="$2"
+            shift
+            ;;
+        *)
+            if [[ "$1" == -* ]] || [[ "${THEME-}" ]]; then
+                echo "unknown option $1"
+                print_usage
+                exit 2
+            fi
+            THEME="$1"
+            ;;
+    esac
+    shift
 done
 
 if [[ -z "${THEME:-}" ]]; then
-  print_usage
+    print_usage
 fi
 
 PATHLIST=(
-  './src/_theme-color.scss'
-  './src/chrome'
-  './src/cinnamon'
-  './src/cinnamon/assets'
-  './src/gnome-shell'
-  './src/gtk-2.0/assets.svg'
-  './src/gtk-2.0/assets-dark.svg'
-  './src/gtk-2.0/gtkrc'
-  './src/gtk-2.0/gtkrc-dark'
-  './src/gtk-2.0/gtkrc-light'
-  './src/gtk-3.0/assets.svg'
-  './src/metacity-1'
-  './src/unity'
-  './src/xfwm4'
+    './src/_theme-color.scss'
+    './src/chrome'
+    './src/cinnamon'
+    './src/cinnamon/assets'
+    './src/gnome-shell'
+    './src/gtk-2.0/assets.svg'
+    './src/gtk-2.0/assets-dark.svg'
+    './src/gtk-2.0/gtkrc'
+    './src/gtk-2.0/gtkrc-dark'
+    './src/gtk-2.0/gtkrc-light'
+    './src/gtk-3.0/assets.svg'
+    './src/metacity-1'
+    './src/unity'
+    './src/xfwm4'
+    './src/qt5ct_palette.conf'
+    './src/qt6ct_palette.conf'
 )
 if [[ -n "${CUSTOM_PATHLIST:-}" ]]; then
-  IFS=', ' read -r -a PATHLIST <<< "${CUSTOM_PATHLIST:-}"
+    IFS=', ' read -r -a PATHLIST <<< "${CUSTOM_PATHLIST:-}"
 fi
 
 EXPORT_QT5CT=0
```

Comment:
i think it was just left here by mistake and never meant to work - i think the right strategy for qt5ct and qt6ct plugins would be using your plugin in the future instead

Reply:
ok i can do that :) its simple enough